### DISCUSSION TITLE
imagemagick import

### DISCUSSION THREAD

**Discussion by ivandiazzz** [xonsh/xonsh#4237] [Q&A] [answered]
Hello! I just started using xonsh and I'm in love. I have however had an issue with the all-important **import** statement. I use imagemagick often and the command to take a screenshot happens to be "import".

If I want to take a screenshot and save it as a png, I would type:

import screenshot.png

and I get the following output:

xonsh: For full traceback set: $XONSH_SHOW_TRACEBACK = True
ModuleNotFoundError: No module named 'screenshot'

has anyone run into this problem before?

**Comment by anki-code** [comment, marked as answer]
Workarounds:
```python
import ./screen.png    # adding `./` tells xonsh to interpret this command as shell command
```
```python
aliases['scr'] = 'import'  # create alias and add it to ~/.xonshrc
scr screen.png

# Also you can avoid typing the name:
aliases['scr'] = 'import screenshot-@($(date -Iseconds).replace(":","").strip()).png'
scr
ls screenshot-*
```
```python
$[import screen.png]    # explicit running the shell command
```
```python
@('import') screen.png  # python substitution for the command name
```

I recommend to read [xonsh-cheatsheet](https://github.com/anki-code/xonsh-cheatsheet/blob/main/README.md).
